src/shogun/regression/GaussianProcessRegression.cpp

```diff
@@ -56,12 +56,14 @@ CRegressionLabels* CGaussianProcessRegression::apply_regression(CFeatures* data)
 			feat=m_method->get_features();
 
 		result=new CRegressionLabels(get_mean_vector(feat));
+		result->put("confidence", get_variance_vector(feat));
```

Comment:
this shouldn't be on by default imo, as it is computationally expensive to compute the confidences. I think a boolean flat would be best.